README.md

```diff
@@ -118,3 +118,11 @@ We recommend users to write their code in TypeScript, especially if their new Ra
 ![JS completion through editor support](img/js-completion.png)
 
 See the example on [examples/js]() for more information also for reusing the unit test suites from JavaScript code.
+
+### Repository commands
+
+ `npm run test` - run unit tests
```

Comment:
maybe have these commands in individual lines? At the moment, they're in one paragraph and may be a little hard to read.